logging/usage_windows.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2019-2020 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package logging
+
+import (
+	"context"
+	"sync"
+	"syscall"
+	"time"
+)
+
+// UsageLogThread utility logging method
+func UsageLogThread(ctx context.Context, log Logger, period time.Duration, wg *sync.WaitGroup) {
```

Comment:
The `UsageLogThread` implementation should be unified for all the platforms. Only the underlying time acquisition portion should be platform dependent.

Reply:
Reimplemented code and process times are retrieved from a function in `utils` also used in network test file.